scripts/travis_ubuntu.sh

```diff
@@ -3,10 +3,18 @@
 ## -x be verbose; write what we are doing, as we do it
 set -ex
 
+# for non-LTS && EOL'ed ubuntu, the mirror url must be modified to run `apt update`
+cmd=ls
+if [ $2 = ":eoan" ]; then
+    cmd="sed -i 's/\(security\|archive\).ubuntu/old-releases.ubuntu/g' /etc/apt/sources.list"
+fi 
+
 sudo docker pull "$1$2"                                                                                           \
 &&                                                                                                                \
 sudo docker create --name mobydick "$1$2" /bin/bash -c                                                            \
-"apt-get    update                                                                                             && \
+"${cmd} &&
+                     \
```

Comment:
@RyanBernX : thanks ! Looks great. TravisCI seems to fail to start on last push (?!) : can you try to remove the above extra empty line to see if CI is triggered OK ?

Reply:
I've added another commit. Could you please review https://github.com/opencollab/arpack-ng/pull/299/commits/3f66e11946271df3f4420e4a01c6f30e06958f20 ?